packages/components/nodes/Xero/xero.ts

```diff
@@ -0,0 +1,425 @@
+    networks?: INodeParams[];
+    inputParameters?: INodeParams[];
+
+    constructor() {
+        this.label = 'Xero';
+        this.name = 'xero';
+        this.icon = 'xero.svg';
+        this.type = 'action';
+        this.version = 1.0;
+        this.description = 'Perform Xero operations';
+        this.incoming = 1;
+        this.outgoing = 1;
+
+        this.actions = [
+            {
+                label: 'Operation',
+                name: 'operation',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Get all invoices',
+                        name: 'getAllInvoices',
+                        description: 'Returns all the invoices from Xero account.'
+                    },
+                    {
+                        label: 'Get single invoice',
+                        name: 'getSingleInvoice',
+                        description: 'Returns single invoice from Xero account.'
+                    },
+                    {
+                        label: 'Create Invoice',
+                        name: 'createInvoice',
+                        description: 'Creates a new draft invoice.'
+                    },
+                    {
+                        label: 'Send email',
+                        name: 'sendToEmail',
+                        description: 'Send email with invoice to primary email.'
+                    }
+                ],
+                default: 'getXeroBalance'
+            }
+        ] as INodeParams[];
+
+        this.credentials = [
+            {
+                label: 'Credential Method',
+                name: 'credentialMethod',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Xero OAuth2',
+                        name: 'xeroOAuth2Api'
+                    }
+                ],
+                default: 'xeroOAuth2Api'
+            }
+        ] as INodeParams[];
+
+        this.inputParameters = [
+            {
+                label: 'Tenant',
+                name: 'tenant',
+                type: 'asyncOptions',
+                loadMethod: 'getTenants'
+            },
+            {
+                label: 'Invoice',
+                name: 'invoice',
+                type: 'asyncOptions',
+                loadMethod: 'getInvoices',
+                show: {
+                    'actions.operation': ['getSingleInvoice', 'sendToEmail']
+                }
+            },
+            // Get existing contacts
+            {
+                label: 'Who do you want to bill?',
+                name: 'contactName',
+                type: 'asyncOptions',
+                loadMethod: 'getContacts',
+                show: {
+                    'actions.operation': ['createInvoice']
+                }
+            },
+            {
+                label: 'Type of invoice',
+                name: 'invoiceType',
+                type: 'string',
```

Comment:
would be better to change it into `options`. So that way user can just select either one, save the time to type those values in